server/service/src/stocktake_line/validate.rs

```diff
@@ -102,26 +101,13 @@ pub fn check_reason_is_valid(
 }
 
 pub fn check_stock_line_reduced_below_zero(
-    connection: &StorageConnection,
-    store_id: &str,
-    stock_line_id: &str,
+    total_number_of_packs: &f64,
     counted_number_of_packs: &f64,
-) -> Result<bool, RepositoryError> {
-    let outbound_shipments = InvoiceLineRepository::new(connection).query_by_filter(
-        InvoiceLineFilter::new()
-            .stock_line_id(EqualFilter::equal_to(&stock_line_id))
-            .store_id(EqualFilter::equal_to(store_id))
-            .invoice_status(InvoiceRowStatus::New.equal_to())
-            .invoice_type(InvoiceRowType::OutboundShipment.equal_to()),
-    )?;
+    available_number_of_packs: &f64,
+) -> bool {
+    let adjustment = total_number_of_packs - counted_number_of_packs;
```

Comment:
There is some about multiple parameters of the same type that makes me nervous (can easily be misused), personally would do `check_stock_line_reduction_below_zero(stock_line: &StockLineRow, counted_number_of_packs: &f64)`